### PR TITLE
ADAPT-000 Add "version:" for gatsby-source-storyblok

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Encountered a Gatsby #10123 Config error since the "version:" was missing. This PR adds "version:" back to to gatsby-source-storyblok config.

# Review By (Date)
- asap?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to...
3. Verify...

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ module.exports = {
         accessToken: process.env.GATSBY_STORYBLOK_ACCESS_TOKEN,
         homeSlug: "home",
         resolveRelations: storyblokRelations,
-        process.env.NODE_ENV === "production" ? "published" : "draft",
+        version: process.env.NODE_ENV === "production" ? "published" : "draft",
```
Comment (rebeccahongsf) [inline gatsby-config.js:65]: Would it make sense to also change `process.env.NODE_ENV` to use `activeEnv` instead? 🤔

**Review by sherakama — APPROVED** [review]
TY

File: gatsby-config.js (hunk 1 above)
Comment (sherakama) [reply gatsby-config.js:65]: Yeah, that is a good point. activeEnv would make sense here. Can you send in a new PR for this so we can test it unlike myself rushing to patch things up and breaking stuff.

Comment (rebeccahongsf) [reply gatsby-config.js:65]: Opened a new PR here: https://github.com/SU-SWS/saa_alumni/pull/54 😄

**Pull request merged**